addon/components/frost-date-time-picker.js

```diff
@@ -48,17 +58,17 @@ export default Component.extend({
   @readOnly
   @computed('_validatedValue')
   _dateValue (_validatedValue) {
-    if (_validatedValue) {
-      return _validatedValue.format(Format.date)
+    if (_validatedValue && _validatedValue.isValid()) {
```

Comment:
I think this logic could be avoided/no-repeated if this `isValid` check was done in `_validatedValue`